modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js

```diff
@@ -38,6 +38,8 @@ const ALIGN_POSITIONS = [
 
 const SELECTOR_TOOLTIP = '.tooltip[role="tooltip"]';
 const SELECTOR_TRIGGER = `
+	.taglib-clay-tag [title],
+	.taglib-clay-tag [data-restore-title],
```

Comment:
I think maybe we finally want to be more aggressive and just do `[title], [data-title]` as a trigger selector and clean up all the specific exceptions we've been adding over time instead of adding a new one.

I don't think the `[data-restore-title]` selector should really be used even though it appears in other entries below. It basically signals an intermediate step where we usually move the `title` attribute to `data-restore-title` so we can leave html the same way we found it before showing a tooltip and avoiding a double tooltip triggered by the default behaviour of a `title` attribute.

Reply:
`[date-restore-title]` was necessary for the mouseout even and removing the tooltip. But I think you are right, we should just do all `[title]` and `[data-title]` and then we can add an explicit opt-out such as `:not([data-disable-tooltip])`